LocalStoreSimulation/src/ro/scit/storeentities/Store.java

```diff
@@ -0,0 +1,48 @@
+/***
+ *
+ * @author Georgiana
+ *
+ */
+
+package ro.scit.storeentities;
+import entities.Product;
+import java.util.ArrayList;
+import java.util.List;
+public class Store {
+
+    static List<SoldItem> orderList = new ArrayList<>();
+    List<Product> productList;
+    List<StockItem> stockList = new ArrayList<>();
+
+    Product product = new Product();
+    SoldItem sold = new SoldItem();
+
+
+    public static boolean sellProduct(List orderList) {
+        SoldItem sold = new SoldItem();
+        Product product = new Product();
+        int id = 0;
+        int stockQuantity= product.getQuantity()- sold.getSoldQuantity();
```

Comment:
you do, new Product and the product.getQuantity... that should return nothing....